packages/arb-rpc-node/cmd/arb-dev-node/arb-dev-node.go

```diff
@@ -96,10 +95,9 @@ func startup() error {
 	walletbalance := fs.Int64("walletbalance", 100, "amount of funds in each wallet (Eth)")
 	arbosPath := fs.String("arbos", "", "ArbOS version")
 	enableFees := fs.Bool("with-fees", false, "Run arbos with fees on")
-	dbDir := fs.String("dbdir", "", "directory to load dev node on. Use tempory if empty")
+	dbDir := fs.String("dbdir", "", "directory to load dev node on. Use temporay if empty")
```

Comment:
Nice catch, but I'm afraid it's actually "temporary" :laughing:

Reply:
🤣